charts/hail-search/templates/deployment.yaml

```diff
@@ -73,11 +75,13 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           volumeMounts:
-            - mountPath: {{ .Values.environment.DATASETS_DIR }}
-              name: datasets
-            - mountPath: {{ .Values.environment.SSD_DATASETS_DIR }}
-              name: ssd-datasets
+            - mountPath: {{ .Values.environment.HAIL_SEARCH_DATA_DIR }}
+              name: hail-search-data-dir
               readOnly: true
+            {{- if .Values.inMemoryVolumeEnabled }}
+            - mountPath: {{ .Values.environment.IN_MEMORY_DIR }}
+              name: in-memory-dir
+            {{- end }}
```

Comment:
Do we also need to add a mount for the `REFERENCE_DATASETS_DIR`

Reply:
I think we actually just need `/seqr` instead of `/seqr/seqr-hail-search-data`.

Reply:
Yeah, this aligns with what I did in [airflow](https://github.com/broadinstitute/seqr-pipeline-airflow/pull/285/files#diff-e02c8035fb33918e7057de3b717c24d2f123f31e383b6558acc2808bb24851c9R104) and how I've made everything else work downstream.  This was a brainfart.